src/api/parser/src/feed/processor.js

```diff
@@ -5,22 +5,23 @@
  */
 
 const Parser = require('rss-parser');
-const { logger, fetch } = require('@senecacdot/satellite');
+const fetch = require('node-fetch');
+const { logger } = require('@senecacdot/satellite');
 
 const Post = require('../data/post');
 const Feed = require('../data/feed');
 const ArticleError = require('../data/article-error');
 
 // Check for cached ETag and Last-Modified info on the feed.
-function hasHeaders(feed) {
+const hasHeaders = (feed) => {
```

Comment:
If the only thing you do in an arrow function is return, you don't need a body: `=> (feed.etag || feed.lastModified)`